src/blockencodings.cpp

```diff
@@ -106,13 +106,12 @@ ReadStatus PartiallyDownloadedBlock::InitData(const CBlockHeaderAndShortTxIDs& c
     std::vector<bool> have_txn(txn_available.size());
     {
```

Comment:
why does 18635 not touch configure.ac like upstream?

Reply:
Resolved in latest push